scripts/version.sh

```diff
@@ -80,7 +80,7 @@ then
 fi
 
 # run checks before creating the new version
-yarn lint
+# yarn lint
```

Comment:
Any reason why the lint and flow steps are commented out? Debugging leftovers?

Reply:
Seems so. I have made necessary changes.